### ISSUE TITLE
MultiDataList filters are applied based on the label field and not it's value

### ISSUE THREAD

**Issue by mrtinkz** [appbaseio/reactivesearch#244]
**Issue Type:**

bug
**Description:**

MultiDataList filters are applied based on the label and not it's value. In the below code snippet when an option is selected, it's label (data collection object label field) is used to apply filters on ResultSet and not it's value, which results in not returning any data. The issue can be replicated by changing the case of Open Source to open Source in the label using this [link](https://codesandbox.io/s/github/appbaseio/reactivesearch/tree/dev/packages/web/examples/MultiDataList?from-embed)

```
<MultiDataList
	title="MultiDataList"
	componentId="CitySensor"
	dataField="group.group_topics.topic_name_raw.raw"
	data={[
	         { label: 'open Source', value: 'Open Source' },
		 { label: 'Social', value: 'Social' },
		 { label: 'Adventure', value: 'Adventure' },
		 { label: 'Music', value: 'Music' },
	]}
/>

```

**Screenshots:**

**Minimal reproduction of the problem with instructions:**

**Reactivesearch version:** 2.2.1

**Browser:** [all | Chrome XX | Firefox XX | Edge XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Anything else:**

**Comment by dhruvdutt** [comment]
@mrtinkz Thanks for the report.
We've identified the issue you're facing and should be resolving it in the next release which will be happening soon.

**Comment by metagrover** [comment]
@dhruvdutt Check this with SingleDataList as well.

**Comment by divyanshu013** [comment]
Thanks for contributing, this will be fixed in the upcoming release v2.3.0 🥂
